### ISSUE TITLE
Cannot sign in to Azure

### ISSUE THREAD

**Issue by steamprodz** [ankitbko/vscode-pull-request-azdo#70]
- Extension version: 0.0.25
- VSCode Version: 1.67.2
- OS: Windows 10

Steps to Reproduce:

1. Open 'Azure Devops' extension sidebar
2. Click 'Sign In'
3. Nothing happens

**Comment by roviralinxea** [comment]
Same Problem here 👋

**Comment by brumhard** [comment]
@steamprodz @roviralinxea I think you need to set `Org Url` and `Project Name` in the settings (`@ext:ankitbko.vscode-pull-request-azdo`). If you click sign in after that you get the dialog to put in your PAT.
That's also described here: https://github.com/ankitbko/vscode-pull-request-azdo#getting-started

**Comment by salvadornava01** [comment]
Same here!

**Comment by salvadornava01** [comment]
> 

You are correct! Thank you buddy!

**Comment by patras1** [comment]
After the Sign-In I have inserted the PAT but I get the following message: "Unable to authenticate. Signout and try again."
with no other indication for the problem. 
I have filled the 'Org Url' and the 'Project Name' correctly and the PAT was generated from azure correctly.
Any advice?

**Comment by patelriki13** [comment]
new baby for VS Code: 
For sing-in (ACtivity Bar)

![image](https://user-images.githubusercontent.com/3973924/192536051-2f8fefda-d0b0-4dcf-9cb4-d48716217b77.png)

**Comment by patras1** [comment]
Still having the some problem...

> After the Sign-In I have inserted the PAT but I get the following message: "Unable to authenticate. Signout and try again." with no other indication for the problem. I have filled the 'Org Url' and the 'Project Name' correctly and the PAT was generated from azure correctly. Any advice?

**Comment by schneim** [comment]
I guess your remote is not named origin?

**Comment by zcoop98** [comment]
Does your Azure logic have multiple tenants by chance? I've had many issue with various Azure extensions stemming from not having the right [Azure tenant set to be my default](https://stackoverflow.com/q/23937228).

**Comment by nonprenom** [comment]
Hi, I have the same issue and I found a beginning of solution...

my repository is at this url : 
https://dev.azure.com/<org name>/<client name>/_git/<project>/

At the beginning, I put : 
Org Url = https://dev.azure.com/<org name>/<client name>/_git/
Project Name = <project>

I was getting the error "Unable to authenticate. Signout and try again."

The doc says Org Url should be https://dev.azure.com/<org_name>
So I put  : 

Org Url = https://dev.azure.com/<org name>/
Project Name = <client name>/_git/<project>

So, now I can login, I can see my git repo with its sub modules but I still can't see the PR. (0 pull requests in this category)

The extension errors is now : 

_"Fetching pull requests failed: A project name is required in order to reference a Git repository by name"_

**Comment by NechiK** [comment]
> Hi, I have the same issue and I found a beginning of solution...
> 
> my repository is at this url : `https://dev.azure.com/<org name>/<client name>/_git/<project>/`
> 
> At the beginning, I put : Org Url = `https://dev.azure.com/<org name>/<client name>/_git/` Project Name = `<project>`
> 
> I was getting the error "Unable to authenticate. Signout and try again."
> 
> The doc says Org Url should be `https://dev.azure.com/<org_name>` So I put :
> 
> Org Url = `https://dev.azure.com/<org name>/` Project Name = `<client name>/_git/<project>`
> 
> So, now I can login, I can see my git repo with its sub modules but I still can't see the PR. (0 pull requests in this category)
> 
> The extension errors is now :
> 
> _"Fetching pull requests failed: A project name is required in order to reference a Git repository by name"_

Yeah, I have the same error. I've tried to enter a project name and a Git repo name. Nothing works.

I've one more question. If I have space in the project name, should I enter it exactly as it's in the URL? Like `Project%20Name`?

**Comment by phuongvo9** [comment]
I have the same issue

**Comment by jsuddsjr** [comment]
So, looks like I need to PAT after all. Any idea what permissions I should grant?
